SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -98,12 +109,15 @@ handle_zip_file() {
 
     local path_to_storage=`handle_netcdf_file $tmp_dir/$nc_file`
 
-    local extracted_file
-    for extracted_file in `find $tmp_dir -type f`; do
-        local file_basename=`basename $extracted_file`
-        s3_put_no_index $extracted_file $path_to_storage/$file_basename
-    done
-
+    if [ -n "$path_to_storage" ];then
+        local extracted_file
+        for extracted_file in `find $tmp_dir -type f`; do
+            local file_basename=`basename $extracted_file`
+            s3_put_no_index $extracted_file $path_to_storage/$file_basename
+        done
+    else
```

Comment:
This should never happen, because `handle_netcdf_file` will have already killed the process instead of returning an empty path, but better safe than sorry I guess?